src/cmds/fs/uniq.c

```diff
@@ -0,0 +1,178 @@
+/**
+ * @file
+ * @brief Filter  adjacent matching lines from FILE, writing to standard output.
+ *
+ * @date 02.07.19
+ * @author Nastya Nizharadze
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <ctype.h>
+#include <unistd.h>
+#include <errno.h>
+
+#define NL_AND_NUL	2
+
+#define FLAG_DUPLICATES 1 << 4
```

Comment:
It's a good idea to add parenthesis:
```
#define FLAG_DUPLICATES (1 << 4)
```
Otherwise it may lead to confusing results, for example
`int a = FLAG_DUPLICATES * 2;`
Without parenthesis:
a = 1 << 4 = 1 << 8 = 256 /* That's probably not what we mean */
With parenthesis:
a = (1 << 4) * 2 = 16 * 2 = 32

Reply:
Done.